lib/kubernetes-deploy/runner.rb

```diff
@@ -290,30 +290,23 @@ def confirm_context_exists
     end
 
     def confirm_namespace_exists
-      _, _, st = run_kubectl("get", "namespace", @namespace, namespaced: false)
+      _, _, st = run_kubectl("get", "namespace", @namespace, with_namespace: false)
```

Comment:
> otherwise developers may assume that the flag accepts the namespace name: run_kubectl("get", "pods", namespace: "mynamespace")

Sure, if it were `namespace`, but it wasn't: it was `namespaced`, which sounds like it would accept a boolean and is still the best option imo.